lib/OpenQA/CLI/archive.pm

```diff
@@ -21,7 +21,8 @@ sub command ($self, @args) {
             url => $url,
             archive => $path,
             'with-thumbnails' => $options{'with-thumbnails'},
-            'asset-size-limit' => $options{'asset-size-limit'}});
+            'asset-size-limit' => $options{'asset-size-limit'},
+            'all' => $options{'all'}});
```

Comment:
```suggestion
            'all' => $options{'all'},
        });
```

Or maybe:

```suggestion
            'all' => $options{'all'},
        }
     );
```

what `./tools/tidyall` do?

Reply:
`tidyall` is fine with that. and I didnt alter it from its original. So lets skip this, if you dont mind.